tailwind.config.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
     },
     container: {
       center: true,
-      padding: '1.25rem',
+      padding: '3.75rem',
```

Comment:
Check if it's working everywhere

Reply:
We don't need to use paddings with rem, let's discuss this